docker-compose.yml

```diff
@@ -74,7 +74,7 @@ services:
     environment:
       API_URL: http://localhost:3000
     ports:
-      - "5000:5000"
+      - "5050:5000"
```

Comment:
hmm... I don't think it's the case? I have tested this changeset both via nginx and by directly accessing.

Above, I am only changing the host port, and nginx config resolves ports from inside the Docker network.

So, the only thing that changes is directly accessing that site by `localhost:5050` instead of `localhost:5000`.

Reply:
alternatively... do we need it exposed at all, or the idea is that we always access it via nginx proxy? 

if you think it would be easier,  I can change the internal port to `5050` too (then default.conf) would need to be updated too.

Reply:
> hmm... I don't think it's the case? I have tested this changeset both via nginx and by directly accessing the service.
>
> Docker syntax is a bit weird here, because the mapping is -$EXTERNAL_PORT:$INTERNAL_PORT.
>
> Above, I am only changing the host port, and nginx config resolves ports from inside the Docker network.

You are absolutely right! I confused the internal and external ports 🙈